lib/tests/streamlit/web/server/server_test_case.py

```diff
@@ -46,7 +46,7 @@ def get_new_ioloop(self) -> IOLoop:
         create a new AsyncIOLoop so that we can access its underlying
         asyncio_loop instance in our `event_loop` property.
         """
-        return AsyncIOLoop()
+        return AsyncIOLoop(make_current=False)
```

Comment:
I don't know why, but all the async server test cases after the first fail with "RuntimeError: Event loop is closed" if this argument is omitted.

Reply:
Hm, it looks like this may be due to creating `AsyncIOLoops` without the `make_current=False` argument being deprecated in tornado 6.2: https://www.tornadoweb.org/en/stable/ioloop.html